libtransmission/session.h

```diff
@@ -404,6 +405,9 @@ struct tr_session
     // Only session.cc should use this.
     int peer_socket_tos_ = *tr_netTosFromName(TR_DEFAULT_PEER_SOCKET_TOS_STR);
 
+    Bandwidth* bandwidthGroupFind(std::string_view name);
+    std::map<std::string, Bandwidth*> bandwidth_groups;
```

Comment:
Looks like these Bandwidths are currently leaked on shutdown.

Does the value type need to be a pointer -- could it just be a Bandwidth object instead? Then the cleanup would happen automatically.